supporter-product-data/src/main/scala/com/gu/conf/ZuoraQuerierConfig.scala

```diff
@@ -55,21 +52,15 @@ object ZuoraQuerierConfig {
       )
     }
 
-  private def getParameterValue(configPath: String, name: String, params: List[Parameter]) =
+  private def findParameterValue(name: String, params: List[Parameter]) =
     params
-      .find(_.getName == configPath + name)
+      .find(_.getName.endsWith(name))
```

Comment:
I had to check there that none are a suffix of another - if we're sure we're only fetching non-recursively, how about `split` it on a dot and check the `last`?